packages/@glimmer/runtime/lib/compat/svg-inner-html-fix.ts

```diff
@@ -30,41 +31,51 @@ export function applySVGInnerHTMLFix(
   let div = document.createElement('div');
 
   return class DOMChangesWithSVGInnerHTMLFix extends DOMClass {
-    insertHTMLBefore(parent: HTMLElement, nextSibling: Node, html: string): Bounds {
+    insertHTMLBefore(
+      parent: Simple.Element,
+      nextSibling: Option<Simple.Node>,
+      html: string
+    ): Bounds {
+      if (html === '') {
+        return super.insertHTMLBefore(parent, nextSibling, html);
+      }
+
       if (parent.namespaceURI !== svgNamespace) {
         return super.insertHTMLBefore(parent, nextSibling, html);
       }
 
-      return fixSVG(parent, div, html, nextSibling);
+      // TODO: why are these casts okay???
```

Comment:
@chadhietala @wycats

Reply:
The mental model is that `DOMChanges` is dealing with live elements. Updating doesn't mean anything in SSR case.